src/main/java/com/scriptbasic/utility/functions/UtilityFunctions.java

```diff
@@ -143,4 +146,19 @@ public static Long length(final Object arg) {
         return null;
     }
 
+    @BasicFunction(classification = Utility.class)
```

Comment:
Great functions and they were really missing from the implementation. I appreciate that you fill in these compatibility holes.